src/main/java/org/example/Person.java

```diff
@@ -127,18 +136,55 @@ public void addSibling(Person person){
             this.siblings.add(person);
         }
 
-        public List<Person> getGrandChildren(Person person){
-            List<Person> grandChildrens = new ArrayList<>();
-            for (Person p: person.children){
+        public List<Person> getGrandChildren(){
+            List<Person> grandChildren = new ArrayList<>();
+            for (Person p: this.children){
                 for (Person gC : p.getChildren()){
-                    grandChildrens.add(gC);
+                    grandChildren.add(gC);
                 }
             }
-            return grandChildrens;
+            return grandChildren;
         }
```

Comment:
Werkt deze manier voor getGrandChildren ook als er nog geen grandchildren in de lijst staan? Of krijg je dan een error? In de uitwerkingen, checken ze eerst of de List geen ''null'' waarde heeft: https://github.com/hogeschoolnovi/backend-java-family-tree-uitwerkingen/blob/main/src/main/java/Person.java

Reply:
Goede opmerking, ik moet het testen